kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java

```diff
@@ -22,7 +22,7 @@
 public class JavaIndexerConfig extends IndexerConfig {
 
   @Parameter(names = "--emit_jvm_signatures", description = "Generate vnames with jvm signatures.")
-  private boolean emitJvmSignatures;
+  protected boolean emitJvmSignatures;
```

Comment:
I would prefer adding a setter below to match the rest of the parameters.

Reply:
Done.